src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const manArr = typeof century !== 'undefined'
+    ? people
+      .filter((person) => person.sex === 'm')
+      .filter((person) => (century === Math.ceil(person.died / 100)))
+
+    : people.filter((person) => person.sex === 'm');
```

Comment:
```suggestion
  const men = people.filter(person =>
    century
      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
      : person.sex === 'm'
  );
```

same but simpler and shorter